### ISSUE TITLE
learning rate and number of epoch

### ISSUE THREAD

**Issue by douxiaotian** [flairNLP/flair#2571]
for training from scratch and fine-tune the model, we both need to tune the hyperparameters like learning rate and number of epoch, are there some hueristics on better hyperparameter tuning? Thanks.

**Comment by whoisjones** [comment]
hi @douxiaotian, in our tutorials, we use the recommended hyperparameters from the respective papers. You can use the recommended parameters from there for your first tries.
